docs/pages/repo/docs/reference/command-line-reference.mdx

```diff
@@ -326,7 +326,8 @@ Will execute _only_ the `test` tasks in each workspace. It will not `build`.
 
 #### `--parallel`
 
-Default `false`. Run commands in parallel across workspaces and ignore the dependency graph. This is useful for developing with live reloading.
+Default `false`. Run commands in parallel across workspaces and ignore the task dependency graph.
+Before you use this option, consider using the [`persistent`](#persistent) config instead.
```

Comment:
I would be harsher here - maybe use a default Callout to direct to persistent and also note that parallel will be deprecated in a future version.

Reply:
updated. i'm not sure if there any use cases for `--parallel` in the wild we haven't considered yet, so i don't want to do a blanket statement yet.